service/server.py

```diff
@@ -119,18 +111,17 @@ def get_official_copy(title_ref):
 @app.route('/title_search_postcode/<postcode>', methods=['GET'])
 def get_properties(postcode):
```

Comment:
We've got very high level unit tests for this applications. Because of that, we cover a relatively small subset of edge cases. We could use the change this story to improve it and write a few proper tests. Off the top of my head (for both postcode and address search):

- server calls es_utils with the right page number
- server does't call db_utils when es_utils returns no results (and returns an empty page)
- server calls db_utils with the list of title numbers from es_utils
- server returns results from db_utils in the right order